package.json

```diff
@@ -665,6 +665,15 @@
           "description": "Path to run neovim executable. For example, /usr/bin/nvim, or  C:\\Program Files\\Neovim\\bin\\nvim.exe",
           "default": "nvim"
         },
+        "vim.enableVimrc": {
+          "type": "boolean",
+          "description": "Use key mappings from a .vimrc file",
+          "default": "true"
+        },
+        "vim.vimrcPath": {
+          "type": "string",
+          "description": "Path to a Vim configuration file. If unset, it will check for $HOME/.vimrc or $HOME/_vimrc"
```

Comment:
Remind me - is `_vimrc` a gVim thing?

Reply:
It might be a gVim thing; I know of it as more generally a Windows thing. Windows hasn't always supported file names that start with `.`, and even now some programs / dialogs don't allow it, so Windows versions of Vim and Vim derivatives tend to support a `_vimrc` file.

Reply:
Gotcha, makes sense